plugin/completion.py

```diff
@@ -25,10 +25,12 @@ def run(self, edit: sublime.Edit, index: int, event: Optional[dict] = None) -> N
         if not detail or not documentation:
             # To make sure that the detail or documentation fields doesn't exist we need to resove the completion item.
             # If those fields appear after the item is resolved we show them in the popup.
-            self.do_resolve(item)
-        else:
-            minihtml_content = self.get_content(documentation, detail)
-            self.show_popup(minihtml_content)
+            session = self.best_session('completionProvider.resolveProvider')
```

Comment:
This is a bit odd. We can only enter LspResolveDocs when either:

1) the session is a resolveProvider, or
2) there is actual documentation

From what I understand, elm-language-server is apparently *not* a resolveProvider? Then we must have checked if there was documentation present in the first in `format_completion` from views.py?

Reply:
Ah, but if there's no detail field then we also enter this block. I think that's the mistake?

Reply:
From the spec about the `completionItem/resolve` request:
> The request can only delay the computation of the detail and documentation properties.
So we have to try resolving if we don't have either of those, I think.

Reply:
> From what I understand, elm-language-server is apparently _not_ a resolveProvider? Then we must have checked if there was documentation present in the first place in `format_completion` from views.py?

Yes, in this case:
 - the server is not a `resolveProvider`
 - there is a `documentation` property filled-in initially